src/main/java/com/shop/dao/impl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,126 @@
+package com.shop.dao.impl;
+
+import com.shop.dao.ManufacturerDao;
+import com.shop.exceptions.DataProcessingException;
+import com.shop.lib.Dao;
+import com.shop.model.Manufacturer;
+import com.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String create = "INSERT INTO manufacturer (manufacturer_name, manufacturer_country) "
+                + "VALUES(?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(create, Statement.RETURN_GENERATED_KEYS);
```

Comment:
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
            PreparedStatement statement = connection
                    .prepareStatement(create, Statement.RETURN_GENERATED_KEYS)) {
```
and then we can remove `statement.close()`